src/collection.js

```diff
@@ -20,15 +25,47 @@ CKEDITOR.define( [ 'emittermixin', 'ckeditorerror', 'utils' ], function( Emitter
 		 * Creates a new Collection instance.
 		 *
 		 * @constructor
+		 * @param {Iterale} [items] Items to be added to the collection.
+		 * @param {String} [idName='id'] The name of the property which is considered to identify an item.
 		 */
-		constructor() {
+		constructor( items, idName ) {
```

Comment:
I've just realised that `add()` cannot accept an iterable collection of items, because it will be impossible to tell whether that object should be added as a single collection item or whether we should iterate over it. That could lead to nasty problems in the future if one wants to create collection of such objects. Let's wait to see how much we'll miss this method of adding items.